### PR TITLE
[Design2-150] DSCO - FontAwesome - Add support of custom icons, update Font Awesome Version

### PR DESCRIPTION
**Pull request by levadadenys**
### [Design2-150] DSCO - FontAwesome - Add support of custom icons, update Font Awesome Version

* updated FontAwesome urls to use Font Awesome v6.6.0
* added $font-awesome-custom-icons-url, which allows us to use our font awesome custom icons

## Links
- jira ticket: [Design2-150](https://codedotorg.atlassian.net/browse/DESIGN2-150)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: shared/css/font.scss [hunk 1]
```diff
@@ -129,12 +129,16 @@ $barlowSemiCondensed-medium: 'Barlow Semi Condensed Medium', $noto-sans-fonts, s
...
-$font-awesome-v4-fonts-url: 'https://dsco.code.org/assets/font-awesome-pro/1684178876/css/v4-font-face.min.css';
-$font-awesome-v4-shims-url: 'https://dsco.code.org/assets/font-awesome-pro/1684178876/css/v4-shims.min.css';
+$font-awesome-core-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/fontawesome.min.css';
+$font-awesome-brands-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/brands.min.css';
+$font-awesome-solid-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/solid.min.css';
+$font-awesome-regular-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/regular.min.css';
+$font-awesome-v4-fonts-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/v4-font-face.min.css';
+$font-awesome-v4-shims-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/v4-shims.min.css';
+$font-awesome-v4-shims-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/v4-shims.min.css';
+$font-awesome-custom-icons-url: 'https://dsco.code.org/assets/font-awesome-pro/1726762595/css/custom-icons.min.css';
```
Comment (molly-moen) [inline shared/css/font.scss:145]: should I update the [duotone url](https://github.com/code-dot-org/code-dot-org/pull/61213/files#diff-7011a1f5c1ddf7f7d7ea73ad715c8c10d134fae6245a7bf407a6ad223b9ffab7R141) to use this new version number? I can also merge it in now and you can update it as part of this pr (either way I think we'll get some easily-resolvable merge conflicts)

Comment (molly-moen) [reply shared/css/font.scss:145]: I merged my pr in but can help resolve conflicts/do a follow-up to update the version!

**Review by molly-moen — APPROVED** [review]

File: shared/css/font.scss (hunk 1 above)
Comment (levadadenys) [reply shared/css/font.scss:145]: No worries, I've just pulled staging and resolved merge conflicts! ✅ 
Re updating url - I've updated in now, just for the future context - different number here stands for different timestamps and versions of font awesome files

**Review by bencodeorg — APPROVED** [review]
LGTM!

**Pull request merged**